test/internet/test-dgram-broadcast-multi-process.js

```diff
@@ -153,7 +153,8 @@ if (process.argv[2] !== 'child') {
               assert.strictEqual(
                 count,
                 messages.length,
-                'A worker received an invalid multicast message'
+                'A worker received an invalid multicast message. ' +
+                `Received ${messages.length} and wanted ${count}.`
```

Comment:
Shouldn't this be `Received ${count} and wanted ${messages.length}.`?
Personally I would just remove the "message" argument and use the default error message.

Reply:
No this is the right way, I asked at the code and learn to make sure. If they don't want the message they can close this PR

Reply:
@nikkistonge I think you might be misunderstanding what @lpinca means in their comment. I think what @lpinca is trying to say is that `messages.length` and `count` should be swapped with each other in the message string added on line 157. It's backwards in the PR as it stands now.

Reply:
Ok swapped them